package.json

```diff
@@ -25,10 +25,10 @@
     "lint": "aegir lint",
     "release": "aegir release",
     "build": "aegir build",
-    "test": "aegir test",
-    "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser",
-    "generate": "bin/generate"
+    "test": "aegir test --node",
+    "test:node": "aegir test --node --target node",
+    "test:browser": "aegir test --node --target browser",
```

Comment:
Without this, stuff breaks with `Error: Cannot find module 'stream'` (and `path` and `os`), because aegir stopped injecting Node globals by default.

@hugomrdias I've  added `--node` to avoid changing anything in tests,
but would like to remove the need for this in a separate PR. Is there an easy fix?